docs/source/ext/yamltohtml.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import yaml
+
+from docutils import nodes
+from docutils.parsers.rst import Directive
+
+
+def get_remote_yaml(url):
+    r = requests.get(url)
+    return yaml.safe_load(r.text)
+
+
+class DaskConfigDirective(Directive):
+    optional_arguments = 3
+
+    def run(self):
+        location, config, schema = self.arguments
+
+        config = get_remote_yaml(config)
+        schema = get_remote_yaml(schema)
+
+        for k in location.split("."):
+            config = config[k]
+            schema = schema["properties"].get(k, {})
+        html = generate_html(config, schema, location)
+        return [nodes.raw("", html, format="html")]
+
+
+def setup(app):
+    app.add_directive("yamltohtml", DaskConfigDirective)
```

Comment:
I might rename this directive to something more namespaced. Perhaps `dask-config-block`, since this is a block directive.